src/Tiles.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+function Tiles ({image, title, text}) {
+    return (
+        <section>
+            <img src={image} />
+                <h2>
+                    {title}
+                </h2>
+                <p>
+                    {text}
+                </p>

```

Comment:
De opdracht was dat je meerdere paragraphen tekst mee moet kunnen geven. Bij jou is er maar 1 paragraaf mogelijk.